boa3/analyser/typeanalyser.py

```diff
@@ -1715,7 +1769,19 @@ def visit_List(self, list_node: ast.List) -> List[Any]:
         :param list_node: the python ast list node
         :return: the value of the list
         """
-        return [self.get_type(value) for value in list_node.elts]
+        get_literal_values = hasattr(list_node, 'get_literal_value') and list_node.get_literal_value
+        result = []
+        for value in list_node.elts:
+            if get_literal_values:
+                item = self.visit(value, get_literal_value=True)
+                if item is None or isinstance(item, ast.AST):
+                    item = self.get_type(value)
```

Comment:
These 3 lines are used on 4 different occasions, do you think it would be better to implement them on a method? Like:
```python
def _get_literal_values_from_value(value):
    element = self.visit(value, get_literal_value=True)
    if element is None or isinstance(element, ast.AST):
        element = self.get_type(value)
    return element
```